### ISSUE TITLE
NetCDF4 dependency is missing in requirements file. 

### ISSUE THREAD

**Issue by bascrezee** [MeteoSwiss-APN/fls_sat_verif#5]
```
(fls_sat_verif) bcrezee@tsa-ln003:/scratch/bcrezee/scripts/support-c2e/src/fls_sat_verif(support-c2e)$ fls_sat_verif --calc_fractions --wd /scratch/bcrezee/wd_fls_sat_verif/ --start 22010100 --e
nd 22010200 --interval 6 --exp_model_dir /store/s83/bcrezee/EXP_TST/560/ --exp 560 --model c2e -vvv 

-------------------------------
Working directory: /scratch/bcrezee/wd_fls_sat_verif/
-------------------------------

INFO:root:Your working directories:
INFO:root:   /scratch/bcrezee/wd_fls_sat_verif/sat
INFO:root:   /scratch/bcrezee/wd_fls_sat_verif/tqc
INFO:root:   /scratch/bcrezee/wd_fls_sat_verif/fls
INFO:root:   /scratch/bcrezee/wd_fls_sat_verif/plots
INFO:root:Calculating FLS fractions 
INFO:root:   for Jan 01, 2022, 00 UTC to Jan 03, 2022, 00 UTC.
WARNING:root:Created new obs dataframe:
WARNING:root:  /scratch/bcrezee/wd_fls_sat_verif/fls/obs.p
WARNING:root:Created new fcst dataframe:
WARNING:root:  /scratch/bcrezee/wd_fls_sat_verif/fls/fcst_560.p
DEBUG:root:SAT timestamp: 2112312345
WARNING:root:SAT file: /scratch/bcrezee/wd_fls_sat_verif/sat/MSG_lscl-cosmo1eqc3km_2112312345_c2e.nc
Traceback (most recent call last):
  File "/scratch/bcrezee/miniconda3/envs/fls_sat_verif/bin/fls_sat_verif", line 33, in <module>
    sys.exit(load_entry_point('fls-sat-verif', 'console_scripts', 'fls_sat_verif')())
  File "/scratch/bcrezee/miniconda3/envs/fls_sat_verif/lib/python3.8/site-packages/click/core.py", line 1130, in __call__
    return self.main(*args, **kwargs)
  File "/scratch/bcrezee/miniconda3/envs/fls_sat_verif/lib/python3.8/site-packages/click/core.py", line 1055, in main
    rv = self.invoke(ctx)
  File "/scratch/bcrezee/miniconda3/envs/fls_sat_verif/lib/python3.8/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/scratch/bcrezee/miniconda3/envs/fls_sat_verif/lib/python3.8/site-packages/click/core.py", line 760, in invoke
    return __callback(*args, **kwargs)
  File "/scratch-shared/meteoswiss/scratch/bcrezee/scripts/support-c2e/src/fls_sat_verif/cli.py", line 189, in main
    obs, fcst = calc_fls_fractions(
  File "/scratch-shared/meteoswiss/scratch/bcrezee/scripts/support-c2e/src/fls_sat_verif/utils.py", line 363, in calc_fls_fractions
    ds = xr.open_dataset(obs_file).squeeze()
  File "/scratch/bcrezee/miniconda3/envs/fls_sat_verif/lib/python3.8/site-packages/xarray/backends/api.py", line 479, in open_dataset
    engine = plugins.guess_engine(filename_or_obj)
  File "/scratch/bcrezee/miniconda3/envs/fls_sat_verif/lib/python3.8/site-packages/xarray/backends/plugins.py", line 148, in guess_engine
    raise ValueError(error_msg)
ValueError: found the following matches with the input file in xarray's IO backends: ['netcdf4', 'h5netcdf', 'scipy']. But their dependencies may not be installed, see:
https://docs.xarray.dev/en/stable/user-guide/io.html 
https://docs.xarray.dev/en/stable/getting-started-guide/installing.html
```

@swester I am looking into this, let me know if you encountered this problem already and know the solution.

**Comment by bascrezee** [comment]
Quick solution: Install netCDF4 by hand:   `conda install -c conda-forge netCDF4`


Will be added to requirements and the fresh installation should be tested.

**Comment by bascrezee** [comment]
Has been fixed in #7
